### PR TITLE
Introduce generic_repr() model decorator, that adds generic __repr__() to an ORM model.

### PR DESCRIPTION
**Pull request by AndrewPashkin**
Subj.

### CODE REVIEW

**Comment by AndrewPashkin** [comment]
@kvesteri, tests are green.

**Comment by kvesteri** [comment]
Thanks for the PR! I think more pythonic way would be to add this as a class decorator (in the same fashion as https://docs.python.org/3.6/library/functools.html#functools.total_ordering).

**Comment by AndrewPashkin** [comment]
@kvesteri, it makes sense.

**Comment by AndrewPashkin** [comment]
@kvesteri, I've updated the PR.

File: docs/models.rst [hunk 1]
```diff
@@ -8,3 +8,11 @@ Timestamp
 .. module:: sqlalchemy_utils.models
 
 .. autoclass:: Timestamp
+
+
+GenericReprMixin
```
Comment (kvesteri) [inline docs/models.rst]: The docs should refer to ``generic_repr``

File: tests/test_models.py [hunk 2]
```diff
@@ -38,3 +37,52 @@ def test_updated(self, session, Article):
         session.commit()
 
         assert article.updated >= then and article.updated <= datetime.utcnow()
+
+
+class TestGenericReprMixin:
```
Comment (kvesteri) [inline tests/test_models.py]: Rename this test case (its testing generic_repr not GenericReprMixin so maybe TestGenericRepr would be good).

**Comment by frol** [comment]
I really like this idea! I hope this will be merged soon.

@AndrewPashkin Could you, please, address the @kvesteri's requests? Those seem to be so small renamings after all.

**Comment by AndrewPashkin** [comment]
@frol, done.

**Comment by AndrewPashkin** [comment]
@kvesteri, weird - I've got a `lint` fail on the file, that is not changed in this PR. I think it's because the linter is updated.

**Comment by frol** [comment]
@kvesteri I am kindly pinging you as I am eager to clean my code using this helper util.

**Comment by kvesteri** [comment]
GJ, now merged!

**Pull request merged**